src/System.Diagnostics.PerformanceCounter/src/System.Diagnostics.PerformanceCounter.csproj

```diff
@@ -0,0 +1,179 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{9758106C-5D4E-475D-B5E7-B7ABC46B1DDA}</ProjectGuid>
+    <RootNamespace>System.Diagnostics.PerformanceCounter</RootNamespace>
+    <AssemblyName>System.Diagnostics.PerformanceCounter</AssemblyName>
+    <DefineConstants>$(DefineConstants);FEATURE_REGISTRY</DefineConstants>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <IsPartialFacadeAssembly Condition="'$(TargetGroup)' == 'netfx'">true</IsPartialFacadeAssembly>
+    <GeneratePlatformNotSupportedAssemblyMessage Condition="'$(TargetsWindows)' != 'true'">SR.PlatformNotSupported_PerfCounters</GeneratePlatformNotSupportedAssemblyMessage>
+    <UWPCompatible>false</UWPCompatible>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(TargetsFreeBSD)' == 'true' OR '$(TargetsUnknownUnix)' == 'true' ">
```

Comment:
Why is this needed?

Reply:
If it remains it should be changed to `<NoWarn>$(NoWarn);0649</NoWarn>`

Reply:
Probably just leftover from copying this csproj from a different one.